### ISSUE TITLE
calicoctl appears to break when specifying etcd

### ISSUE THREAD

**Issue by chuckbutler** [Metaswitch/calico-docker#133]
When kicking off calicoctl w/ the linked binary in the docs: wget https://github.com/Metaswitch/calico-docker/releases/download/v0.0.6/calicoctl

I get some additional feedback from the binary when providing an external etcd server. It appears the python code is expecting another argument that was not passed:

```
ubuntu@ip-10-35-160-172:~$ sudo ./calicoctl master --ip=10.35.160.72 --etcd=10.230.43.76
Traceback (most recent call last):
  File "<string>", line 513, in <module>
  File "<string>", line 333, in master
  File "<string>", line 79, in __init__
ValueError: need more than 1 value to unpack
```

**Comment by chuckbutler** [comment]
Upon further investigation, it appears this binary linked in the docs is quite old, and this may be resolved with a quick PR to the doc.

**Comment by tomdee** [comment]
@robbrockbank Can you update the docs to point at Alex's special "latest" URL (it's in the video he recorded here http://www.projectcalico.org/getting-started/docker/)

@chuckbutler Does an ENV var work for you or do you need the option of passing on the CLI?

**Comment by lxpollitt** [comment]
@robbrockbank We should include both http://projectcalico.org/latest/calicoctl and an example of getting a specific release from github (as today).  You should flag that projectcalico.org/latest/calicoctl is not HA, so may not be suitable for production orchestration systems.

**Comment by robbrockbank** [comment]
Documentation has been updated.  Closing issue.

Fixed in #140 and #144

**Comment by chuckbutler** [comment]
Sorry about the delay. the ENV Var did indeed work for me, when setting ETCD_AUTHORITY things behaved as expected.
